LZ4 requires access to java.nio.Buffer raw address field (after upgrade to 2.6.0)

### Environment
- JDK version: 16

### Error logs

```
java.lang.ExceptionInInitializerError
	at com.github.housepower.repackaged.io.airlift.compress.lz4.Lz4RawCompressor.encodeRunLength(Lz4RawCompressor.java:297)
	at com.github.housepower.repackaged.io.airlift.compress.lz4.Lz4RawCompressor.emitLastLiteral(Lz4RawCompressor.java:275)
	at com.github.housepower.repackaged.io.airlift.compress.lz4.Lz4RawCompressor.compress(Lz4RawCompressor.java:98)
	at com.github.housepower.repackaged.io.airlift.compress.lz4.Lz4Compressor.compress(Lz4Compressor.java:44)
	at com.github.housepower.buffer.CompressedBuffedWriter.flushToTarget(CompressedBuffedWriter.java:79)
	at com.github.housepower.serde.BinarySerializer.maybeDisableCompressed(BinarySerializer.java:123)
	at com.github.housepower.protocol.DataRequest.writeImpl(DataRequest.java:50)
	at com.github.housepower.protocol.Request.writeTo(Request.java:30)
	at com.github.housepower.protocol.QueryRequest.writeImpl(QueryRequest.java:85)
	at com.github.housepower.protocol.Request.writeTo(Request.java:30)
	at com.github.housepower.client.NativeClient.sendRequest(NativeClient.java:169)
	at com.github.housepower.client.NativeClient.sendQuery(NativeClient.java:163)
	at com.github.housepower.client.NativeClient.sendQuery(NativeClient.java:116)
	at com.github.housepower.jdbc.ClickHouseConnection.sendQueryRequest(ClickHouseConnection.java:270)
	at com.github.housepower.jdbc.statement.ClickHouseStatement.lambda$executeUpdate$0(ClickHouseStatement.java:92)
	at com.github.housepower.misc.ExceptionUtil.rethrowSQLException(ExceptionUtil.java:76)
	at com.github.housepower.jdbc.statement.ClickHouseStatement.executeUpdate(ClickHouseStatement.java:75)
	at com.github.housepower.jdbc.statement.ClickHouseStatement.executeQuery(ClickHouseStatement.java:100)
	at com.github.housepower.jdbc.statement.ClickHouseStatement.execute(ClickHouseStatement.java:69)
	at com.github.housepower.jdbc.statement.ClickHousePreparedQueryStatement.execute(ClickHousePreparedQueryStatement.java:67)
	at com.zaxxer.hikari.pool.ProxyPreparedStatement.execute(ProxyPreparedStatement.java:44)
	at com.zaxxer.hikari.pool.HikariProxyPreparedStatement.execute(HikariProxyPreparedStatement.java)
	at io.vertx.jdbcclient.impl.actions.JDBCPreparedQuery.execute(JDBCPreparedQuery.java:73)
	at io.vertx.jdbcclient.impl.actions.JDBCPreparedQuery.execute(JDBCPreparedQuery.java:39)
	at io.vertx.ext.jdbc.impl.JDBCConnectionImpl.lambda$schedule$3(JDBCConnectionImpl.java:219)
	at io.vertx.core.impl.ContextImpl.lambda$null$0(ContextImpl.java:159)
	at io.vertx.core.impl.AbstractContext.dispatch(AbstractContext.java:100)
	at io.vertx.core.impl.ContextImpl.lambda$executeBlocking$1(ContextImpl.java:157)
	at io.vertx.core.impl.TaskQueue.run(TaskQueue.java:76)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.base/java.lang.Thread.run(Thread.java:831)
Caused by: com.github.housepower.repackaged.io.airlift.compress.IncompatibleJvmException: LZ4 requires access to java.nio.Buffer raw address field
	at app//com.github.housepower.repackaged.io.airlift.compress.lz4.UnsafeUtil.<clinit>(UnsafeUtil.java:53)
	... 33 more
```

### Steps to reproduce
implementation 'com.github.housepower:clickhouse-native-jdbc-shaded:2.6.0'


### Other descriptions
2.5.6 version - no issue

--- a/pom.xml
+++ b/pom.xml
@@ -100,7 +100,7 @@
         <maven.release.plugin.version>2.5.3</maven.release.plugin.version>
         <maven.checkstyle.plugin.version>3.1.1</maven.checkstyle.plugin.version>
 
-        <aircompressor.version>0.18</aircompressor.version>
+        <aircompressor.version>0.21</aircompressor.version>
         <google.jsr305.version>3.0.2</google.jsr305.version>
         <jmh.version>1.27</jmh.version>
         <yandex-clickhouse-jdbc.version>0.2.5</yandex-clickhouse-jdbc.version>

